spot_rl_experiments/spot_rl/envs/skill_manager.py

```diff
@@ -199,21 +209,24 @@ def __init_spot(
             else open_close_drawer_config
         )
 
-        self.open_close_drawer_config = (
-            construct_config_for_open_close_drawer()
-            if not open_close_drawer_config
-            else open_close_drawer_config
-        )
+        # self.open_close_drawer_config = (
```

Comment:
why do we want to remove this?

Reply:
It seems to be duplicate of code above it no?

Reply:
You are right, Kavit! Then remove these comments entirely please. Thanks in advance!

Reply:
This is addressed.